(node:5672) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:5672) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:5672) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:5508) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:5508) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:5508) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:5508) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:5508) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:5508) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:5508) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:1492) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:1492) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:1492) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:1492) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:1492) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:1492) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:1492) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:5784) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:5784) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:5784) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:5784) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:5784) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:5784) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:5784) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:2388) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:2388) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:2388) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:2388) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:2388) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:2388) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:2388) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:12696) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:12696) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:12696) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:12696) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:12696) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:12696) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:12696) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:2352) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:2352) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:2352) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:2352) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:2352) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:2352) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:2352) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:7940) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:7940) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:7940) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:7940) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:7940) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:7940) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:7940) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:12732) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:12732) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:12732) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:12732) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at runNextTicks (internal/process/task_queues.js:66:3)
    at processImmediate (internal/timers.js:429:9)
(node:12732) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:12732) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at runNextTicks (internal/process/task_queues.js:66:3)
    at processImmediate (internal/timers.js:429:9)
(node:12732) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:12732) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:12732) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 4)
(node:12732) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:12732) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 5)
(node:10376) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:10376) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:10376) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:10376) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:10376) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:10376) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at runNextTicks (internal/process/task_queues.js:66:3)
    at processImmediate (internal/timers.js:429:9)
(node:10376) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:10376) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:10376) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 4)
(node:10376) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:10376) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 5)
(node:7684) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at runNextTicks (internal/process/task_queues.js:66:3)
    at processImmediate (internal/timers.js:429:9)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:7684) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:7684) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:7684) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:7684) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:7684) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:7684) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:7684) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:7684) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 4)
(node:7684) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:7684) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 5)
(node:8500) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at runNextTicks (internal/process/task_queues.js:66:3)
    at processImmediate (internal/timers.js:429:9)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:8500) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:8500) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:8500) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:8500) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:8500) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:8500) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:8500) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:8500) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 4)
(node:8500) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:8500) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 5)
(node:11656) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:11656) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:11656) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:11656) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:11656) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:5876) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at runNextTicks (internal/process/task_queues.js:66:3)
    at processImmediate (internal/timers.js:429:9)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:5876) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:5876) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:5876) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:5876) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:5876) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:5876) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:5876) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:5876) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 4)
(node:5876) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:5876) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 5)
(node:10488) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at runNextTicks (internal/process/task_queues.js:66:3)
    at processImmediate (internal/timers.js:429:9)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:10488) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:10488) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:10488) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:10488) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:10488) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:10488) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:10488) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:10488) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 4)
(node:10488) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:10488) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 5)
(node:6488) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at runNextTicks (internal/process/task_queues.js:66:3)
    at processImmediate (internal/timers.js:429:9)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:6488) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:6488) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:6488) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:6488) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:6488) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:6488) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:6488) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:6488) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 4)
(node:6488) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:6488) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 5)
(node:5560) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at runNextTicks (internal/process/task_queues.js:66:3)
    at processImmediate (internal/timers.js:429:9)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:5560) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:5560) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:5560) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:5560) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:5560) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:5560) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:5560) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:5560) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 4)
(node:5560) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:5560) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 5)
(node:5836) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at runNextTicks (internal/process/task_queues.js:66:3)
    at processImmediate (internal/timers.js:429:9)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:5836) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:5836) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:5836) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:5836) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:5836) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:5836) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:5836) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:5836) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 4)
(node:5836) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:5836) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 5)
(node:5032) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at runNextTicks (internal/process/task_queues.js:66:3)
    at processImmediate (internal/timers.js:429:9)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:5032) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:5032) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:5032) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:5032) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:5032) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:5032) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:5032) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:5032) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 4)
(node:5032) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:5032) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 5)
(node:2964) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:2964) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:2964) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:2964) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:2964) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:2964) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:2964) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:2964) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:2964) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 4)
(node:2964) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:2964) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 5)
(node:7280) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at runNextTicks (internal/process/task_queues.js:66:3)
    at processImmediate (internal/timers.js:429:9)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:7280) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:7280) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:7280) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:7280) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:7280) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:7280) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:7280) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:7280) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 4)
(node:7280) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:7280) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 5)
(node:3312) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:3312) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:3312) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:3312) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:3312) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:3312) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:3312) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:3312) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:3312) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 4)
(node:3312) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:3312) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 5)
(node:11160) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:11160) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:11160) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:11160) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:11160) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:11160) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:11160) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:11160) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:11160) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 4)
(node:11160) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:11160) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 5)
(node:5460) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:5460) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:5460) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:5460) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:5460) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:5460) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:5460) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:5460) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:5460) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 4)
(node:5460) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:5460) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 5)
(node:2916) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:2916) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:2916) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:2916) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:2916) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:2916) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:2916) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:2916) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:2916) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 4)
(node:2916) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:2916) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 5)
(node:11048) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at runNextTicks (internal/process/task_queues.js:66:3)
    at processImmediate (internal/timers.js:429:9)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:11048) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:11048) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:11048) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:11048) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:11048) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:11048) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:11048) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:11048) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 4)
(node:11048) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:11048) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 5)
(node:548) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:548) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:548) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:548) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:548) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:548) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:548) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:548) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:548) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 4)
(node:548) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:548) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 5)
(node:10380) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at runNextTicks (internal/process/task_queues.js:66:3)
    at processImmediate (internal/timers.js:429:9)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:10380) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:10380) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:10380) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:10380) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:10380) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:10380) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:10380) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:10380) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 4)
(node:10380) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:10380) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 5)
(node:1388) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:1388) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:1388) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:1388) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:1388) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:1388) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:1388) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:1388) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:1388) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 4)
(node:1388) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at D:\Project\NitroDBT\sync\datasync.js:34:34
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:1388) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 5)
